README.md

```diff
@@ -24,11 +24,39 @@ anonlink is repsonsible for garbling the PII so that it can be de-identified pri
 
 ### Installing with an existing Python install
 
-1. Download the tools as a zip file using the "Clone or download" button on GitHub.
+
+#### Cloning the Repository
+
+Clone the project locally as a Git repository
+```shell
+git clone https://github.com/mitre/data-owner-tools.git
+```
+
+Or download as a zip file:
+
+1. [Click this link to download the project as a zip](https://github.com/mitre/data-owner-tools/archive/refs/heads/master.zip) or use the "Clone or download" button on GitHub.
 1. Unzip the file.
-1. From the unzipped directory run:
 
-    `pip install -r requirements.txt`
+#### Set up a virtual environment _(Optional, but recommended)_
+
+It can be helpful to set up a virtual environment to isolate project dependencies from system dependencies.
+There are a few libraries that can do this, but this documentation will stick with `venv` since that is included
+in the Python Standard Library.
+
+```shell
+# Navigate to the project folder
+cd data-owner-tools/
+# Create a virtual environment in a `venv/` folder
+python3 -m venv venv/
```

Comment:
Minor thing, but we have tended to just mention the executable for python as just `python`.